components/automate-cds/cmd/automate-cds/commands/serve.go

```diff
@@ -0,0 +1,72 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
```

Comment:
For new stuff, we could try to use the stdlib errors handling. It's improved lately and offers the same functionality as this package.

The equivalent of
```go
return errors.Wrapf(err, "some %s went bananas", x)
```
would be
```go
return fmt.Errorf("some %s went bananas: %w", x, err)
```